src/app/collection/pages/collection-ncyte/footer/footer.component.html

```diff
@@ -0,0 +1,22 @@
+<section class = "footer">
+    <div class = "learn-more">
+           This is the page for the ncyte curriculum collection on 
+           CLARK (
+           <aria-label class = "link" href="http://clark.center"> www.clark.center  </aria-label>
```

Comment:
I don't think there is a aria-label tag, this looks like it should be a 'a' tag with a aria-label attribute.